### DISCUSSION TITLE
Error during import a repository

### DISCUSSION THREAD

**Discussion by Amazingmonkeys** [community/community#16928] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to import a repo from another git link, I tried to import through "import repository", but it failed:

We found an error during import
We recommend restarting the import process, but if you are still having trouble [contact support](https://github.com/contact).
 There was an error pushing commits to GitHub.

**Comment by lucalves** [comment, marked as answer]
Hello @Amazingmonkeys, how are you?

I think [this post](https://github.community/t/there-was-an-error-pushing-commits-to-github-for-lfs-enabled-repo/176661/2) by @[canuckjacq](https://github.com/canuckjacq) can help you. 

Could you take a look?
